test/juttle-service-bin.spec.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+var path = require('path');
+var expect = require('chai').expect;
+var child_process = require('child_process');
+var Promise = require('bluebird');
+var findFreePort = Promise.promisify(require('find-free-port'));
+var service = require('../lib/juttle-service');
+
+let juttle_service_cmd = path.resolve(`${__dirname}/../bin/juttle-service`);
+let juttle_service_client_cmd = path.resolve(`${__dirname}/../bin/juttle-service-client`);
+
+describe('juttle-service-client binary unit tests', function() {
```

Comment:
This is a style nit, but I prefer shorter describe and it descriptions, and to have them be done in such a way that the nested describes plus the it reads as a reasonable description.

As written this would print "juttle-service-client binary unit tests Can execute juttle-service-client binary with --help".

I'd prefer this to be simply `describe('juttle-service-client')` and `it('can run with --help')`